diff --git a/sources/mainwindow.cpp b/sources/mainwindow.cpp
index 5a99857..3b0b5c0 100644
--- a/sources/mainwindow.cpp
+++ b/sources/mainwindow.cpp
@@ -2143,7 +2143,7 @@ void MainWindow::on_about_triggered()
                                                       + tr("<li>Printing support using CUPS %1</li>").arg(CUPS_VERSION)
 #endif // WITH_CUPS
                                                       + tr("</ul>"
-                                                           "<p>See <a href=\"https://launchpad.net/qpdfview\">launchpad.net/qpdfview</a> for more information.</p><p>&copy; 2012-2016 The qpdfview developers</p>")));
+                                                           "<p>See <a href=\"https://launchpad.net/qpdfview\">launchpad.net/qpdfview</a> for more information.</p><p>&copy; 2012-2017 The qpdfview developers</p>")));
 }
 
 void MainWindow::on_focusCurrentPage_activated()
diff --git a/translations/qpdfview_ast.ts b/translations/qpdfview_ast.ts
index b176170..f57538c 100644
--- a/translations/qpdfview_ast.ts
+++ b/translations/qpdfview_ast.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vea &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pa más información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 Los desendolcadores de qpdfview&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vea &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pa más información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Los desendolcadores de qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_az.ts b/translations/qpdfview_az.ts
index 6202d9e..86b4993 100644
--- a/translations/qpdfview_az.ts
+++ b/translations/qpdfview_az.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Daha geniş məlumat üçün &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; bax.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 The qpdfview proqramçıları.&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Daha geniş məlumat üçün &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; bax.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview proqramçıları.&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_be.ts b/translations/qpdfview_be.ts
index 03adeb0..96e19d6 100644
--- a/translations/qpdfview_be.ts
+++ b/translations/qpdfview_be.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_bg.ts b/translations/qpdfview_bg.ts
index 6ed1c7d..bf39450 100644
--- a/translations/qpdfview_bg.ts
+++ b/translations/qpdfview_bg.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Вижте &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; за повече информация.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 The qpdfview developers&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Вижте &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; за повече информация.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_bs.ts b/translations/qpdfview_bs.ts
index e9791c7..2508d5f 100644
--- a/translations/qpdfview_bs.ts
+++ b/translations/qpdfview_bs.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_ca.ts b/translations/qpdfview_ca.ts
index 8eedf44..57cb2b6 100644
--- a/translations/qpdfview_ca.ts
+++ b/translations/qpdfview_ca.ts
@@ -683,8 +683,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Mire &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; per a mé sinformació.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview developers&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Mire &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; per a mé sinformació.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_cs.ts b/translations/qpdfview_cs.ts
index c885dee..420ce0f 100644
--- a/translations/qpdfview_cs.ts
+++ b/translations/qpdfview_cs.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Na další informace se podívejte na &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Vývojáři qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Na další informace se podívejte na &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Vývojáři qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_da.ts b/translations/qpdfview_da.ts
index d1beb6d..5759bb5 100644
--- a/translations/qpdfview_da.ts
+++ b/translations/qpdfview_da.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_de.ts b/translations/qpdfview_de.ts
index 7da4db3..7cbdfd2 100644
--- a/translations/qpdfview_de.ts
+++ b/translations/qpdfview_de.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Siehe &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; für weitere Informationen.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Die qpdfview-Entwickler&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Siehe &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; für weitere Informationen.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Die qpdfview-Entwickler&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_el.ts b/translations/qpdfview_el.ts
index 2e288d3..18e0499 100644
--- a/translations/qpdfview_el.ts
+++ b/translations/qpdfview_el.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_en_GB.ts b/translations/qpdfview_en_GB.ts
index 8d22a5a..cd470e7 100644
--- a/translations/qpdfview_en_GB.ts
+++ b/translations/qpdfview_en_GB.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview developers&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_eo.ts b/translations/qpdfview_eo.ts
index ba3b0b6..9726bf1 100644
--- a/translations/qpdfview_eo.ts
+++ b/translations/qpdfview_eo.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; por plua informo.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2013 La programantoj de qpdfview&lt;/p&gt; {2012-2014 ?} {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; por plua informo.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 La programantoj de qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_es.ts b/translations/qpdfview_es.ts
index c760171..de1c91e 100644
--- a/translations/qpdfview_es.ts
+++ b/translations/qpdfview_es.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vea &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para mas información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 El equipo desarrollador de qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vea &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para mas información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 El equipo desarrollador de qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_eu.ts b/translations/qpdfview_eu.ts
index 16ba8e3..56b3499 100644
--- a/translations/qpdfview_eu.ts
+++ b/translations/qpdfview_eu.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_fi.ts b/translations/qpdfview_fi.ts
index 7377b9f..b61bc1c 100644
--- a/translations/qpdfview_fi.ts
+++ b/translations/qpdfview_fi.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_fr.ts b/translations/qpdfview_fr.ts
index 74fd6ac..5ce0b74 100644
--- a/translations/qpdfview_fr.ts
+++ b/translations/qpdfview_fr.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Voir &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pour plus de renseignements.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Les développeurs de qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Voir &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pour plus de renseignements.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Les développeurs de qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_gl.ts b/translations/qpdfview_gl.ts
index 2ab8bc9..b4ca63f 100644
--- a/translations/qpdfview_gl.ts
+++ b/translations/qpdfview_gl.ts
@@ -683,8 +683,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Ver &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para máis información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Os desenvolvedores de qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Ver &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para máis información.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Os desenvolvedores de qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_he.ts b/translations/qpdfview_he.ts
index 25518bb..226f72a 100644
--- a/translations/qpdfview_he.ts
+++ b/translations/qpdfview_he.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_hr.ts b/translations/qpdfview_hr.ts
index 11dcbb3..b954146 100644
--- a/translations/qpdfview_hr.ts
+++ b/translations/qpdfview_hr.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Pogledajte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; za više informacija.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2013 Qpdfview razvojni programeri&lt;/p&gt; {2012-2014 ?} {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Pogledajte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; za više informacija.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Qpdfview razvojni programeri&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_hu.ts b/translations/qpdfview_hu.ts
index 750a652..a7a8c6e 100644
--- a/translations/qpdfview_hu.ts
+++ b/translations/qpdfview_hu.ts
@@ -673,7 +673,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_id.ts b/translations/qpdfview_id.ts
index bbb27fd..c5d129a 100644
--- a/translations/qpdfview_id.ts
+++ b/translations/qpdfview_id.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Lihat &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; untuk keterangan lebih jauh.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 Para pengembang qpdfview&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Lihat &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; untuk keterangan lebih jauh.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Para pengembang qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_it.ts b/translations/qpdfview_it.ts
index 74ca1a0..70186f7 100644
--- a/translations/qpdfview_it.ts
+++ b/translations/qpdfview_it.ts
@@ -683,8 +683,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Visitare &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; per ulteriori informazioni.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview developers&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Visitare &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; per ulteriori informazioni.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_ja.ts b/translations/qpdfview_ja.ts
index cc60ed3..ae3bf89 100644
--- a/translations/qpdfview_ja.ts
+++ b/translations/qpdfview_ja.ts
@@ -1055,9 +1055,9 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="1893"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <comment>MainWindow|</comment>
-        <translation>&lt;/ul&gt;&lt;p&gt;詳しくは&lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; を参照。&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview開発陣&lt;/p&gt;</translation>
+        <translation>&lt;/ul&gt;&lt;p&gt;詳しくは&lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; を参照。&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview開発陣&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2608"/>
diff --git a/translations/qpdfview_kk.ts b/translations/qpdfview_kk.ts
index aac4aba..5729b0c 100644
--- a/translations/qpdfview_kk.ts
+++ b/translations/qpdfview_kk.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Көбірек білу үшін &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; қараңыз.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 qpdfview өндірушілері&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Көбірек білу үшін &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; қараңыз.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 qpdfview өндірушілері&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2230"/>
diff --git a/translations/qpdfview_ko.ts b/translations/qpdfview_ko.ts
index 0674156..b399edd 100644
--- a/translations/qpdfview_ko.ts
+++ b/translations/qpdfview_ko.ts
@@ -305,7 +305,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="1578"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2013 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <comment>MainWindow|</comment>
         <translation type="unfinished"></translation>
     </message>
diff --git a/translations/qpdfview_ky.ts b/translations/qpdfview_ky.ts
index 76fcfb7..d0d3bc2 100644
--- a/translations/qpdfview_ky.ts
+++ b/translations/qpdfview_ky.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_lt.ts b/translations/qpdfview_lt.ts
index 1936fff..c08ffe0 100644
--- a/translations/qpdfview_lt.ts
+++ b/translations/qpdfview_lt.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Išsamesnei informacijai, žiūrėkite &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 qpdfview kūrėjai&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Išsamesnei informacijai, žiūrėkite &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 qpdfview kūrėjai&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_ms.ts b/translations/qpdfview_ms.ts
index 3a99a29..6e8962b 100644
--- a/translations/qpdfview_ms.ts
+++ b/translations/qpdfview_ms.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Rujuk &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; untuk maklumat lanjut.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Pembangun qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Rujuk &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; untuk maklumat lanjut.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Pembangun qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_my.ts b/translations/qpdfview_my.ts
index 7905c68..89feaf7 100644
--- a/translations/qpdfview_my.ts
+++ b/translations/qpdfview_my.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_pl.ts b/translations/qpdfview_pl.ts
index d49cae2..0fbcf4c 100644
--- a/translations/qpdfview_pl.ts
+++ b/translations/qpdfview_pl.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Zobacz &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; celem uzyskania więcej informacji.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Programiści qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Zobacz &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; celem uzyskania więcej informacji.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Programiści qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_pt.ts b/translations/qpdfview_pt.ts
index 887c10e..6ab00b7 100644
--- a/translations/qpdfview_pt.ts
+++ b/translations/qpdfview_pt.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Consulte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para obter mais informações.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 The qpdfview developers&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Consulte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para obter mais informações.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_pt_BR.ts b/translations/qpdfview_pt_BR.ts
index ef447eb..8b9396f 100644
--- a/translations/qpdfview_pt_BR.ts
+++ b/translations/qpdfview_pt_BR.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Consulte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para mais informações.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Desenvolvedores do qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Consulte &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; para mais informações.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Desenvolvedores do qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_ro.ts b/translations/qpdfview_ro.ts
index de815b5..1b1d67a 100644
--- a/translations/qpdfview_ro.ts
+++ b/translations/qpdfview_ro.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vizitați  &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; Pentru mai multe informații.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 Dezvoltatorii qpdfview&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Vizitați  &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; Pentru mai multe informații.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Dezvoltatorii qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_ru.ts b/translations/qpdfview_ru.ts
index ff0e7c9..f413ff1 100644
--- a/translations/qpdfview_ru.ts
+++ b/translations/qpdfview_ru.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;См. дополнительную информацию на &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Разработчики qpdfview&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;См. дополнительную информацию на &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Разработчики qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_sk.ts b/translations/qpdfview_sk.ts
index cc3923b..88ce050 100644
--- a/translations/qpdfview_sk.ts
+++ b/translations/qpdfview_sk.ts
@@ -683,8 +683,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Pozrite &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pre ďalšie informácie .&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2015 Vývojári qpdfview&lt;/p&gt;&lt;p&gt;Do slovenčiny preložil DAG Software &lt;i&gt;(Ďanovský Ján)&lt;/i&gt;&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Pozrite &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; pre ďalšie informácie .&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Vývojári qpdfview&lt;/p&gt;&lt;p&gt;Do slovenčiny preložil DAG Software &lt;i&gt;(Ďanovský Ján)&lt;/i&gt;&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_sv.ts b/translations/qpdfview_sv.ts
index 9c632b5..fc48a0f 100644
--- a/translations/qpdfview_sv.ts
+++ b/translations/qpdfview_sv.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Se &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; för ytterligare information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 Utvecklarna av qpdfview&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Se &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; för ytterligare information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Utvecklarna av qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_th.ts b/translations/qpdfview_th.ts
index 64446df..452afd5 100644
--- a/translations/qpdfview_th.ts
+++ b/translations/qpdfview_th.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_tr.ts b/translations/qpdfview_tr.ts
index a9b0170..89047e6 100644
--- a/translations/qpdfview_tr.ts
+++ b/translations/qpdfview_tr.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_ug.ts b/translations/qpdfview_ug.ts
index 1856199..ff118f5 100644
--- a/translations/qpdfview_ug.ts
+++ b/translations/qpdfview_ug.ts
@@ -612,7 +612,7 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
diff --git a/translations/qpdfview_uk.ts b/translations/qpdfview_uk.ts
index 637a8f7..fee978f 100644
--- a/translations/qpdfview_uk.ts
+++ b/translations/qpdfview_uk.ts
@@ -673,8 +673,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Щоб дізнатися більше, відвідайте сторінку &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; Розробники qpdfview, 2012–2015&lt;/p&gt; {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Щоб дізнатися більше, відвідайте сторінку &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;&amp;copy; Розробники qpdfview, 2012–2017&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
diff --git a/translations/qpdfview_vi.ts b/translations/qpdfview_vi.ts
index ed19e3d..76e508d 100644
--- a/translations/qpdfview_vi.ts
+++ b/translations/qpdfview_vi.ts
@@ -612,8 +612,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Xem tại &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; để biết thêm thông tin.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 Nhóm phát triển qpdfview&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;Xem tại &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; để biết thêm thông tin.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 Nhóm phát triển qpdfview&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2231"/>
diff --git a/translations/qpdfview_zh_CN.ts b/translations/qpdfview_zh_CN.ts
index 25cdd49..d1ddf58 100644
--- a/translations/qpdfview_zh_CN.ts
+++ b/translations/qpdfview_zh_CN.ts
@@ -683,8 +683,8 @@
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2026"/>
-        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2016 The qpdfview developers&lt;/p&gt;</source>
-        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;转到 &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; 获取更多信息。&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2014 qpdfview 开发者&lt;/p&gt; {2012-2015 ?} {2012-2016 ?}</translation>
+        <source>&lt;/ul&gt;&lt;p&gt;See &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; for more information.&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 The qpdfview developers&lt;/p&gt;</source>
+        <translation type="unfinished">&lt;/ul&gt;&lt;p&gt;转到 &lt;a href=&quot;https://launchpad.net/qpdfview&quot;&gt;launchpad.net/qpdfview&lt;/a&gt; 获取更多信息。&lt;/p&gt;&lt;p&gt;&amp;copy; 2012-2017 qpdfview 开发者&lt;/p&gt;</translation>
     </message>
     <message>
         <location filename="../sources/mainwindow.cpp" line="2907"/>
