--- /dev/null
+++ b/data_qrc_gen.sh
@@ -0,0 +1,13 @@
+#!/bin/sh
+echo "<RCC>" > data.qrc
+echo " <qresource>" >> data.qrc
+for f in help/help*.html ; do
+  echo "  <file alias=\"$(basename $f)\">$f</file>" >> data.qrc
+done
+rm -f translations/*.qm
+lrelease translations/*.ts
+for f in translations/*.qm ; do
+  echo "  <file alias=\"data/$(basename $f)\">$f</file>" >> data.qrc
+done
+echo " </qresource>" >> data.qrc
+echo "</RCC>" >> data.qrc
--- a/qpdfview.pri
+++ b/qpdfview.pri
@@ -1,3 +1,6 @@
+APPLICATION_VERSION = 0.4.17
+RESOURCES += data.qrc
+
 isEmpty(APPLICATION_VERSION):APPLICATION_VERSION = 0.4.17.99
 
 isEmpty(TARGET_INSTALL_PATH):TARGET_INSTALL_PATH = /usr/bin
--- a/sources/helpdialog.cpp
+++ b/sources/helpdialog.cpp
@@ -44,11 +44,11 @@ HelpDialog::HelpDialog(QWidget* parent) : QDialog(parent)
     m_textBrowser->setSearchPaths(QStringList() << QDir(QApplication::applicationDirPath()).filePath("data") << DATA_INSTALL_PATH);
 
     //: Please replace by file name of localized help if available, e.g. "help_fr.html".
-    m_textBrowser->setSource(QUrl::fromLocalFile(tr("help.html")));
+    m_textBrowser->setSource(QUrl::fromLocalFile(":" + tr("help.html")));
 
     if(m_textBrowser->document()->isEmpty())
     {
-        m_textBrowser->setSource(QUrl::fromLocalFile("help.html"));
+        m_textBrowser->setSource(QUrl::fromLocalFile(":help.html"));
     }
 
     m_dialogButtonBox = new QDialogButtonBox(QDialogButtonBox::Close, Qt::Horizontal, this);
--- a/sources/main.cpp
+++ b/sources/main.cpp
@@ -112,14 +112,14 @@ void loadTranslators()
 
     if(toolkitTranslator->load(QLocale::system(), "qt", "_", QLibraryInfo::location(QLibraryInfo::TranslationsPath))) { qApp->installTranslator(toolkitTranslator); }
 
-    if(applicationTranslator->load(QLocale::system(), "qpdfview", "_", QDir(QApplication::applicationDirPath()).filePath("data"))) { qApp->installTranslator(applicationTranslator); }
+    if(applicationTranslator->load(QLocale::system(), "qpdfview", "_", QDir(QApplication::applicationDirPath()).filePath(":data"))) { qApp->installTranslator(applicationTranslator); }
     else if(applicationTranslator->load(QLocale::system(), "qpdfview", "_", DATA_INSTALL_PATH)) { qApp->installTranslator(applicationTranslator); }
 
 #else
 
     if(toolkitTranslator->load("qt_" + QLocale::system().name(), QLibraryInfo::location(QLibraryInfo::TranslationsPath))) { qApp->installTranslator(toolkitTranslator); }
 
-    if(applicationTranslator->load("qpdfview_" + QLocale::system().name(), QDir(QApplication::applicationDirPath()).filePath("data"))) { qApp->installTranslator(applicationTranslator); }
+    if(applicationTranslator->load("qpdfview_" + QLocale::system().name(), QDir(QApplication::applicationDirPath()).filePath(":data"))) { qApp->installTranslator(applicationTranslator); }
     else if(applicationTranslator->load("qpdfview_" + QLocale::system().name(), DATA_INSTALL_PATH)) { qApp->installTranslator(applicationTranslator); }
 
 #endif // QT_VERSION
