diff --git a/application.pro b/application.pro
index f44a770..15f9315 100644
--- a/application.pro
+++ b/application.pro
@@ -107,7 +107,7 @@ plugin_resolve_all {
     DEFINES += POPPLER_VERSION=\\\"$${POPPLER_VERSION}\\\"
 
     static_pdf_plugin {
-        isEmpty(PDF_PLUGIN_NAME):PDF_PLUGIN_NAME = libqpdfview_pdf.a
+        isEmpty(PDF_PLUGIN_NAME):PDF_PLUGIN_NAME = $${QMAKE_PREFIX_STATICLIB}qpdfview_pdf.$${QMAKE_EXTENSION_STATICLIB}
 
         DEFINES += STATIC_PDF_PLUGIN
         LIBS += $$PDF_PLUGIN_NAME $$PDF_PLUGIN_LIBS
@@ -120,7 +120,7 @@ plugin_resolve_all {
             PKGCONFIG += poppler-qt$${QT_MAJOR_VERSION}
         }
     } else {
-        isEmpty(PDF_PLUGIN_NAME):PDF_PLUGIN_NAME = libqpdfview_pdf.so
+        isEmpty(PDF_PLUGIN_NAME):PDF_PLUGIN_NAME = $${QMAKE_PREFIX_SHLIB}qpdfview_pdf.$${QMAKE_EXTENSION_SHLIB}
     }
 
     DEFINES += PDF_PLUGIN_NAME=\\\"$${PDF_PLUGIN_NAME}\\\"
@@ -133,7 +133,7 @@ plugin_resolve_all {
     DEFINES += LIBSPECTRE_VERSION=\\\"$${LIBSPECTRE_VERSION}\\\"
 
     static_ps_plugin {
-        isEmpty(PS_PLUGIN_NAME):PS_PLUGIN_NAME = libqpdfview_ps.a
+        isEmpty(PS_PLUGIN_NAME):PS_PLUGIN_NAME = $${QMAKE_PREFIX_STATICLIB}qpdfview_ps.$${QMAKE_EXTENSION_STATICLIB}
 
         DEFINES += STATIC_PS_PLUGIN
         LIBS += $$PS_PLUGIN_NAME $$PS_PLUGIN_LIBS
@@ -144,7 +144,7 @@ plugin_resolve_all {
             PKGCONFIG += libspectre
         }
     } else {
-        isEmpty(PS_PLUGIN_NAME):PS_PLUGIN_NAME = libqpdfview_ps.so
+        isEmpty(PS_PLUGIN_NAME):PS_PLUGIN_NAME = $${QMAKE_PREFIX_SHLIB}qpdfview_ps.$${QMAKE_EXTENSION_SHLIB}
     }
 
     DEFINES += PS_PLUGIN_NAME=\\\"$${PS_PLUGIN_NAME}\\\"
@@ -157,7 +157,7 @@ plugin_resolve_all {
     DEFINES += DJVULIBRE_VERSION=\\\"$${DJVULIBRE_VERSION}\\\"
 
     static_djvu_plugin {
-        isEmpty(DJVU_PLUGIN_NAME):DJVU_PLUGIN_NAME = libqpdfview_djvu.a
+        isEmpty(DJVU_PLUGIN_NAME):DJVU_PLUGIN_NAME = $${QMAKE_PREFIX_STATICLIB}qpdfview_djvu.$${QMAKE_EXTENSION_STATICLIB}
 
         DEFINES += STATIC_DJVU_PLUGIN
         LIBS += $$DJVU_PLUGIN_NAME $$DJVU_PLUGIN_LIBS
@@ -168,7 +168,7 @@ plugin_resolve_all {
             PKGCONFIG += ddjvuapi
         }
     } else {
-        isEmpty(DJVU_PLUGIN_NAME):DJVU_PLUGIN_NAME = libqpdfview_djvu.so
+        isEmpty(DJVU_PLUGIN_NAME):DJVU_PLUGIN_NAME = $${QMAKE_PREFIX_SHLIB}qpdfview_djvu.$${QMAKE_EXTENSION_SHLIB}
     }
 
     DEFINES += DJVU_PLUGIN_NAME=\\\"$${DJVU_PLUGIN_NAME}\\\"
@@ -180,7 +180,7 @@ with_fitz {
     DEFINES += FITZ_VERSION=\\\"$${FITZ_VERSION}\\\"
 
     static_fitz_plugin {
-        isEmpty(FITZ_PLUGIN_NAME):FITZ_PLUGIN_NAME = libqpdfview_fitz.a
+        isEmpty(FITZ_PLUGIN_NAME):FITZ_PLUGIN_NAME = $${QMAKE_PREFIX_STATICLIB}qpdfview_fitz.$${QMAKE_EXTENSION_STATICLIB}
 
         DEFINES += STATIC_FITZ_PLUGIN
         LIBS += $$FITZ_PLUGIN_NAME $$FITZ_PLUGIN_LIBS
@@ -192,7 +192,7 @@ with_fitz {
             LIBS += $$FITZ_PLUGIN_LIBS
         }
     } else {
-        isEmpty(FITZ_PLUGIN_NAME):FITZ_PLUGIN_NAME = libqpdfview_fitz.so
+        isEmpty(FITZ_PLUGIN_NAME):FITZ_PLUGIN_NAME = $${QMAKE_PREFIX_SHLIB}qpdfview_fitz.$${QMAKE_EXTENSION_SHLIB}
     }
 
     DEFINES += FITZ_PLUGIN_NAME=\\\"$${FITZ_PLUGIN_NAME}\\\"
@@ -202,14 +202,14 @@ with_fitz {
     DEFINES += WITH_IMAGE
 
     static_image_plugin {
-        isEmpty(IMAGE_PLUGIN_NAME):IMAGE_PLUGIN_NAME = libqpdfview_image.a
+        isEmpty(IMAGE_PLUGIN_NAME):IMAGE_PLUGIN_NAME = $${QMAKE_PREFIX_STATICLIB}qpdfview_image.$${QMAKE_EXTENSION_STATICLIB}
 
         DEFINES += STATIC_IMAGE_PLUGIN
         LIBS += $$IMAGE_PLUGIN_NAME $$IMAGE_PLUGIN_LIBS
         PRE_TARGETDEPS += $$IMAGE_PLUGIN_NAME
     }
     else {
-        isEmpty(IMAGE_PLUGIN_NAME):IMAGE_PLUGIN_NAME = libqpdfview_image.so
+        isEmpty(IMAGE_PLUGIN_NAME):IMAGE_PLUGIN_NAME = $${QMAKE_PREFIX_SHLIB}qpdfview_image.$${QMAKE_EXTENSION_SHLIB}
     }
 
     DEFINES += IMAGE_PLUGIN_NAME=\\\"$${IMAGE_PLUGIN_NAME}\\\"
